src/services/webclient.ts

```diff
@@ -175,7 +175,7 @@ export class WebClientService {
     // State handling
     private startupPromise: ng.IDeferred<{}> = null; // TODO: deferred type
     private startupDone: boolean = false;
-    private pendingInitializationStepRoutines: threema.InitializationStepRoutine[] = [];
+    private pendingInitializationStepRoutines: Set<threema.InitializationStepRoutine> = new Set();
```

Comment:
This change removes the property that the order of the callbacks is not retained. But I don't think that matters for us, so that's fine :slightly_smiling_face: